cxx-qt-gen/src/parser/signals.rs

```diff
@@ -0,0 +1,145 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use crate::parser::parameter::ParserParameter;
+use crate::syntax::fields::fields_named_to_ident_type;
+use syn::{Ident, ItemEnum, Result};
+
+/// Describes an individual Signal
+pub struct ParsedSignal {
+    /// The name of the signal
+    pub ident: Ident,
+    /// The parameters of the signal
+    pub parameters: Vec<ParserParameter>,
```

Comment:
Shouldn't this be a `ParsedParameter`, not a `ParserParameter`?

Reply:
Oops, yup :-)